From 1b760b6c5393b2137410f539d068a84bae1c31ff Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C4=90o=C3=A0n=20Tr=E1=BA=A7n=20C=C3=B4ng=20Danh?=
 <congdanhqx@gmail.com>
Date: Wed, 21 Apr 2021 19:23:43 +0700
Subject: [PATCH] freetype_render_ui_text: use uint8_t instead of u_int8_t

uint8_t is a standard type, while u_int8_t isn't. And we're assigning to
an uint8_t anyway.
---
 kitty/freetype_render_ui_text.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kitty/freetype_render_ui_text.c b/kitty/freetype_render_ui_text.c
index 8c8743b8..f22284f4 100644
--- a/kitty/freetype_render_ui_text.c
+++ b/kitty/freetype_render_ui_text.c
@@ -504,7 +504,7 @@ render_line(PyObject *self UNUSED, PyObject *args, PyObject *kw) {
     if (!PyArg_ParseTupleAndKeywords(args, kw, "|sIIzppkkffI", (char**)kwlist, &text, &width, &height, &family, &bold, &italic, &fg, &bg, &x_offset, &y_offset, &right_margin)) return NULL;
     PyObject *ans = PyBytes_FromStringAndSize(NULL, (Py_ssize_t)width * height * 4);
     if (!ans) return NULL;
-    uint8_t *buffer = (u_int8_t*) PyBytes_AS_STRING(ans);
+    uint8_t *buffer = (uint8_t*) PyBytes_AS_STRING(ans);
     RenderCtx *ctx = (RenderCtx*)create_freetype_render_context(family, bold, italic);
     if (!ctx) return NULL;
     if (!render_single_line((FreeTypeRenderCtx)ctx, text, 3 * height / 4, 0, 0xffffffff, buffer, width, height, x_offset, y_offset, right_margin)) {
-- 
2.31.1.500.gbc6bbdd36b

